2024-10-29 14:12:26,506 [ERROR] __main__: Error during cleanup: 'StorageManager' object has no attribute 'cleanup'
